docs/full_guide.md

```diff
@@ -222,6 +222,34 @@ commit-msg:
 
 When you try to commit `git commit -m "haha bad commit text"` script `template_checker` will be executed. Since commit text doesn't match the described pattern the commit process will be interrupted.
 
+## Bash script example with Commitlint
+
+Let's create a bash script to check conventional commit status `.lefthook/commit-msg/commitlint.sh`:
+
+```bash
+INPUT_FILE=$1
+START_LINE=`head -n1 $INPUT_FILE`
+ERROR=`echo $START_LINE | npx commitlint --color`
+if ! [[ $? == 0 ]]; then
+  echo "${ERROR}"
+  exit 1
+fi
```

Comment:
What is the purpose of this `if` block? I believe that if script consists of this single line:
```sh
echo $(head -n1 $1) | npx commitlint --color
```
it will behave the same (printing error and exiting with non-zero)

Reply:
Indeed, you are right. My bash skills are not great 😅
Updated.